contents/docs/tutorials/b2b.md

```diff
@@ -0,0 +1,247 @@
+
+> **Note:** For an in-depth walkthrough of our Feature Flags functionality, check out the [dedicated tutorial](/docs/tutorials/feature-flags).
+
+If you're unaware, feature flags are a PostHog feature that allows you to safely deploy and roll back new features. It means you can deploy features and then slowly roll them out to your users. If something has gone wrong, you can roll back new features without having to re-deploy your application.
+
+Feature flags make sense especially for B2B businesses because they allow you to roll out features by company, in order to meet the specific use-cases of your clients. For example, a company might need a feature quickly and be willing to be a Beta tester for it, 
+while another company might be more conservative and want to always be using a stable version of your product.
+
+As such, you can roll out certain features to some companies first, as well as roll them back easily if there's ever an issue.
+
+You can then see how the new feature impacted your key retention, conversion, and aggregate usage metrics, in order to determine if it should be rolled out to all users or not. 
+
+At PostHog, we rolled out our session recording feature using feature flags.
+
+### Your Key Metrics in One Place
+
+Rather than flipping back and forth between tabs, you probably want to have your key metrics in one place, so you can quickly get an overview of how your product is performing. 
+
+What should be in each of your dashboards is entirely up to you, and they could look something like this:
+
+![Demo Dashboard Screenshot](../../images/tutorials/b2b/demo-dashboard.png)
+
+Any chart, table, funnel, and graph from our 'Insights' can be added to a dashboard. To do so, once you are happy with a certain view you created, just click 'Add to Dashboard' on the top-right corner. This will let you select what dashboard you want to add the panel to. 
+
+> **Pro Tip:** We found it very useful to create a dashboard to represent AARRR metrics - Acquisition, Activation, Retention, Referral, and Revenue. You need to determine how to track each of these metrics for your product, but it provides a great overview of performance and is a good first step for setting context-specific KPIs. 
+
+Dashboards can be customized as you wish - you can change the color, size, and name of panels, as well as you can update their content whenever you want. You are also able to pin dashboards that you especially care about to the sidebar, as well as create a shareable link for each dashboard. These links can be used to share dashboards across teams, such as between Sales and Marketing teams within your company.
+
+> **Note:** If you want to learn more about dashboards, check out our [dedicated Dashboards page](/docs/features/dashboards).
+
+### Keep track of new signups
+
+If you want to stay on top of new signups (or any other specific action), you can use webhooks to send messages to platforms like Slack and Microsoft Teams.
```

Comment:
"There are several use cases for this:

* help your sales team spot new sign ups that could be prospects
* help your customer success team monitor customers' product usage so they know who needs training
* help your marketing team monitor the quality of sign ups from a new campaign in real time"